recipes/sesh/recipe.yaml

```
@@ -0,0 +1,48 @@
+context:
+  name: sesh
+  version: "2.15.0"
+
+package:
+  name: ${{ name }}
```

```suggestion
  name: sesh
```
You don't need jinja here either.